recipes/gau2grid/meta.yaml

```diff
@@ -0,0 +1,141 @@
+{% set name = "gau2grid" %}
+{% set version = "2.0.5" %}
+{% set sha256 = "b6534bfd2b8b469f66d98d030701ce96e61ded84c1ba521b95f897101d3a5832" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dgasmith/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  binary_relocation: true
+  skip: true  # [py2k]
+
+requirements:
+  build:
+    - cmake >=3.0
+    - {{ compiler('c') }}
+    - ninja  # [win]
+  host:
+    - numpy
+    - python
+
+outputs:
+  - name: gau2grid
+    build:
+      skip: true  # [not py==38]
+      run_exports:
+        - {{ pin_subpackage('gau2grid', max_pin='x') }}
+#      ignore_run_exports:
+#        # for C library, these are purely build tools, not run deps
+#        - numpy
+#        - python
+#        - python_abi
+    requirements:
+      build:
+        - cmake >=3.0
+        - {{ compiler('c') }}
+        - ninja  # [win]
+      host:
+        - numpy
```

Comment:
I guess this is also a runtime dependency? If so https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy